docs/core/linux-prerequisites.md

```diff
@@ -20,25 +20,28 @@ This article shows the dependencies needed to develop .NET Core applications on
 * [Command-line with your favorite editor](tutorials/using-with-xplat-cli.md)
 * [Visual Studio Code](https://code.visualstudio.com/)
 
+> [!NOTE]
+> The .NET Core SDK package is not required for production servers/environments. Only the .NET Core runtime package is needed for apps deployed to production environments. The .NET Core runtime is deployed with apps as part of a self-contained deployment, however, it must be deployed for Framework-dependent deployed apps seperately. For more information about framework-dependent and self-contained deployment types, see [.NET Core application deployment](https://docs.microsoft.com/en-us/dotnet/core/deploying/). Also see [Self-contained Linux applications](https://github.com/dotnet/core/blob/master/Documentation/self-contained-linux-apps.md) for specific guidelines.
```

Comment:
relative link to the deployment topic: `[./deploying/index.md]`

typo: seperately

Reply:
Good catch!